frontend/src/App.css

```diff
@@ -35,20 +35,25 @@ h1 {
     display: inline-block;
     padding: 5px;
     text-shadow:
-            2px 2px 10px var(--background-color),
-            -2px 2px 10px var(--background-color),
-            2px -2px 10px var(--background-color),
-            -2px -2px 10px var(--background-color);
+             5px  5px 9px var(--background-color, #FFFFFF),
+            -5px  5px 9px var(--background-color, #FFFFFF),
+             5px -5px 9px var(--background-color, #FFFFFF),
+            -5px -5px 9px var(--background-color, #FFFFFF);
 }
 
 nav {
     margin-bottom: 1em;
+    padding: 0.2em;
+    border: 1px solid var(--border-color);
```

Comment:
Du benutzt hier em und px vermischt. Good practice wäre es sich da auf eine Einheit zu einigen.

Reply:
Prinzipiell hast Du Recht.
Aber gerade Borders sind ja nur so'ne Linien-Konstrukte und die würde ich tatsächlich gern überall 1 px breit haben.

An anderen Stellen haben sich dann wirklich mal aus Gewohnheit Pixel-Angaben untergemischt, die da nichts zu suchen haben, weil ich eigentlich alles in `em` bemaßen will.

Fazit:
Bei den Borders werd ich's so lassen. Ansonsten muss ich die CSS mal durchforsten, ob sich da noch Pixel-Angaben reingemogelt haben.